conf/scalardb-cluster-custom-values-direct-kubernetes-mode.yaml

```diff
@@ -56,33 +56,21 @@ scalardbCluster:
       cpu: 2000m
       memory: 4Gi
 
-  tolerations:
-    - effect: NoSchedule
-      key: scalar-labs.com/dedicated-node
-      operator: Equal
-      value: scalardb-cluster
-
   affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-          - matchExpressions:
-              - key: scalar-labs.com/dedicated-node
-                operator: In
-                values:
-                  - scalardb-cluster
     podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-              - key: app.kubernetes.io/name
-                operator: In
-                values:
-                  - scalardb-cluster
-              - key: app.kubernetes.io/app
-                operator: In
-                values:
-                  - scalardb-cluster
-          topologyKey: kubernetes.io/hostname
```

Comment:
Ditto. This `requiredDuringSchedulingIgnoredDuringExecution` configuration was used for the old too strict recommendations (`1 ScalarDB Cluster pod on 1 worker node`) as well. We can remove this configuration from this sample config file.